### PR TITLE
Better readme

### PR DESCRIPTION
**Pull request by gggeek**
More features, reword some phrases, remove a dead link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,29 +31,44 @@ The eZ Publish Proprietary License gives you the right to use the source code
...
+- Multimedia support with automatic image conversion and scaling
+- RSS feeds
+- Contact forms
+- Built in webshop
+- Flexible workflow management system
+- Full support for Unicode
+- Template engine
+- A REST API
 - Database abstraction layer
+- MVC architecture
```
Comment (andrerom) [inline README.md]: well.. :)

File: README.md [hunk 2]
```diff
@@ -31,29 +31,44 @@ The eZ Publish Proprietary License gives you the right to use the source code
...
+- Content approval and scheduled publication
+- Multi-site support
+- Multimedia support with automatic image conversion and scaling
+- RSS feeds
+- Contact forms
+- Built in webshop
+- Flexible workflow management system
+- Full support for Unicode
+- Template engine
+- A REST API
```
Comment (andrerom) [inline README.md]: readonly should maybe be mentioned in context of legacy

File: README.md (hunk 1 above)
Comment (gggeek) [inline README.md]: Well, it definitely is. It uses kinda funny naming and breaks the philosophy with templates, but you got 2-level controllers, the model (ezco and friends) and the presentation(templates)

File: README.md (hunk 2 above)
Comment (gggeek) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (andrerom) [inline README.md]: you are right mister salesman.. :)

**Comment by yannickroger** [comment]
+1

**Pull request merged**